daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/TestConfig.scala

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.daml.lf.engine.script
+
+import java.io.File
+import java.time.Duration
+
+import com.digitalasset.platform.services.time.TimeProviderType
+
+case class TestConfig(
+    darPath: File,
+    ledgerHost: Option[String],
+    ledgerPort: Option[Int],
+    participantConfig: Option[File],
+    timeProviderType: TimeProviderType,
+    commandTtl: Duration,
+)
+
+object TestConfig {
+  private val parser = new scopt.OptionParser[TestConfig]("test-script") {
+    head("test-script")
+
+    opt[File]("dar")
+      .required()
+      .action((f, c) => c.copy(darPath = f))
+      .text("Path to the dar file containing the script")
+
+    opt[String]("ledger-host")
+      .optional()
+      .action((t, c) => c.copy(ledgerHost = Some(t)))
+      .text("Ledger hostname")
+
+    opt[Int]("ledger-port")
+      .optional()
+      .action((t, c) => c.copy(ledgerPort = Some(t)))
+      .text("Ledger port")
+
+    opt[File]("participant-config")
+      .optional()
+      .action((t, c) => c.copy(participantConfig = Some(t)))
+      .text("File containing the participant configuration in JSON format")
+
+    opt[Unit]('w', "wall-clock-time")
+      .action { (t, c) =>
+        c.copy(timeProviderType = TimeProviderType.WallClock)
+      }
+      .text("Use wall clock time (UTC). When not provided, static time is used.")
+
+    opt[Long]("ttl")
+      .action { (t, c) =>
+        c.copy(commandTtl = Duration.ofSeconds(t))
+      }
+      .text("TTL in seconds used for commands emitted by the trigger. Defaults to 30s.")
+
+    checkConfig(c => {
+      // TODO: Start in-memory ledger automatically. See #3687.
```

Comment:
Is this comment out of sync with the code below? It looks like you do have code to start an in-memory ledger automatically.

Reply:
Good catch, yes, that comment was out of date.